provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/practitionerrole/PractitionerRoleController.java

```diff
@@ -49,69 +53,51 @@
 
   private PractitionerRole.Bundle bundle(
       MultiValueMap<String, String> parameters, int page, int count) {
-    PractitionerRoleWrapper root = search(parameters);
+    Pair<List<PractitionerRoleWrapper>, Integer> root = search(parameters);
     LinkConfig linkConfig =
         LinkConfig.builder()
             .path("PractitionerRole")
             .queryParams(parameters)
             .page(page)
             .recordsPerPage(count)
-            .totalRecords(1)
+            .totalRecords(root.getRight())
             .build();
     return bundler.bundle(
         BundleContext.of(
             linkConfig,
-            Collections.singletonList(root),
+            root.getLeft(),
             transformer,
             PractitionerRole.Entry::new,
             PractitionerRole.Bundle::new));
   }
 
+  /** Builds the PractitionerWrapper and returns it. */
+  private PractitionerRoleWrapper practitionerRoleWrapperBuilder(
+      ProviderContactsResponse providerContactsResponse,
+      ProviderResponse.Value providerResponse,
+      ProviderSpecialtiesResponse providerSpecialtiesResponse) {
+    PractitionerRoleWrapper.PractitionerRoleWrapperBuilder filteredResults =
+        new PractitionerRoleWrapper.PractitionerRoleWrapperBuilder();
+    filteredResults
+        .providerResponse(ProviderResponse.builder().value(singletonList(providerResponse)).build())
+        .providerContactsResponse(providerContactsResponse)
+        .providerSpecialtiesResponse(providerSpecialtiesResponse);
+    return filteredResults.build();
+  }
```

Comment:
This can be done in one line....Also not sure having it separated out like this buys us anything

i.e.
return new PractitionerWrapper.PractitionerWrapperBuilder()
        .providerResponse(ProviderResponse.builder().value(singletonList(providerResponse)).build())
        .providerContactsResponse(providerContactsResponse).build();